testing/trino-testing/src/main/java/io/trino/testing/TestingTrinoClient.java

```diff
@@ -326,11 +326,11 @@ private static Object convertToRowValue(Type type, Object value)
         }
         if (type.getBaseName().equals("Geometry")) {
             //noinspection RedundantCast
-            return (byte[]) value;
+            return (String) value;
```

Comment:
How does a change of a testing framework affect the data representation in the protocol?

Reply:
It does not. This should've been a separate commit in this PR, but I must have squashed it with the other one by mistake. The problem was that the test was broken but it wasn't running in the CI.

Reply:
I confirm at 427  tag the geo tests were not passing and become passing with this change.

Do you know how long was it broken? 
it seems we had `return (byte[]) value;` "for ever" here and i think geo tests used to be passing with it in older releases.

Reply:
I'm not sure. I checked a few releases back and they were not passing either, but didn't keep digging to see when that happened.

Reply:
In Trino 400, `TestGeoSpatialQueries` works.
In 405 it was probably already not executing on CI, but stil works.
In 410 the test fails with class cast exception here.